criu/cr-restore.c

```diff
@@ -3128,6 +3128,24 @@ static void rst_reloc_creds(struct thread_restore_args *thread_args,
 	thread_args->creds_args = args;
 }
 
+static bool groups_are_okay(gid_t* groups, int n_groups) {
+	int check_num_groups, gids_len;
```

Comment:
https://www.kernel.org/doc/html/v4.10/process/coding-style.html#naming

Given that you've extracted into a function, I do think going back to `n, len` is a good idea.